website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,192 @@
+
+!!!
+
+#### Search
+
+If enabled, search allows the user to filter results based on a query.
+
+![Search enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-search.png =836x*)
+
+#### CustomContent
+
+If enabled, custom content will occupy this space.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-custom-content.png =836x*)
+
+!!! Dont
+
+We recommend against placing heavy UI elements in custom content as it can draw attention away from the list options and will only be visible when the menu is open. Consider placing them in outside the list relative to the trigger for better visibility.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-dont-visually-heavy.png =836x*)
+
+!!!
+
+### AfterOptions
+
+AfterOptions allows for either an HDS standardized footer or custom content to be displayed after the options list. Applicable to both Single and Multiple variants, although the footer has additional functionality in the Multiple variant.
+
+#### Footer
+
+By default, the footer displays below the list and contains content and/or buttons (depending on the variant) that provide additional context (or functions) to the list options.
+
+##### Single
+
+The footer shows a total count, which is customizable to fit the context.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-single-variant-footer-with-counter.png)
+
+##### Multiple
+
+In the multiple select variant, the `afterOptions` and `footer` display buttons to expedite the deselection process and optionally a counter (selected / total).
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-multiple-variant-footer.png)
+
+“Show selected” will change the list to display only selected list items while toggling the label of the button to “Show all.”
+
+![Flow showing the behavior of the show selected button and how it toggles the list to only display selected list items](/assets/components/form/super-select/super-select-flow-select-all.png)
+
+If “Show selected” is used when no selections have been made, a message will appear to let the user know.
+
+![Flow showing the behavior of the show selected button when no selections have been made yet.](/assets/components/form/super-select/super-select-flow-no-selections-select-all.png)
+
+Additionally, “Clear selected” will wipe out the selections and revert the Super Select to its deselected state. This button is hidden if there are no selections.
```

Comment:
Maybe consider this (could be either default or deselected, both kind of make sense in the context):
```suggestion
Additionally, `Clear selected` will revert the Super Select to its default/deselected state. This button is hidden if no options are selected.
```

Reply:
Will make the other changes locally.